tests/conftest.py

```diff
@@ -336,7 +336,7 @@ def pytest_configure(config):
 
 def pytest_runtest_setup(item):
     partest = int(os.environ.get('DEVITO_MPI', 0))
-    if item.get_marker("parallel") and not partest:
+    if item.get_closest_marker("parallel") and not partest:
```

Comment:
wtf pytest does this. I'm also concerned that "closest" may break things if "parallel" doesn't appear as innermost decorator.

Reply:
Only one that works, `get_marker` is deprecated

https://docs.pytest.org/en/latest/mark.html?highlight=get_marker